### ISSUE TITLE
set product image is not working

### ISSUE THREAD

**Issue by prisuri** [woocommerce/woocommerce#27425]
Please provide us with the information requested in this bug report. Without these details, we won't be able to fully evaluate this issue. 
Bug reports lacking detail, or for any other reason than to report a bug, may be closed without action.

**Prerequisites (mark completed items with an [x]):**
- [ ] I have have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**

**Expected behavior**

**Actual behavior**
A clear and concise description of what actually happens. Please be as descriptive as possible; 

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:

**Screenshots**

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
Please select “Get system report”, then “Copy for support”, and then paste it here.
```
</details>

**Comment by juliaamosova** [comment]
Hi @prisuri,

Please provide us with more details about the issue as per the bug report template. Without it, we won't be able to do much about it.

**Comment by prisuri** [comment]
ok when ever i am try to add new product i am facing issue because earlier
it use to came like "earlier product.PNG" now it is coming like "product
upload.PNG"
can u please help me with this as i am not able to add new products

**Comment by tammullen** [comment]
Hi @prisuri 

Sorry for the delay in replying

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only.

The type of issue you submitted looks like a support request which may or may not reveal a bug once proper troubleshooting is done. In order to confirm the bug, please follow one of the steps below:

- Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to see if the solutions listed there apply to your case;
- If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
- Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

If you confirm the bug, please provide us with clear steps to reproduce it.

I am closing this issue for now as it seems to be a support request and not a bug. If I missed something, please leave a comment and we will take a second look.

**Comment by AliaMori** [comment]
I faced the same problem. Went through the above problem solving steps. Didn't help.  Afterwards, I updated to the latest available PHP version (in my case 7.4). Worked wonderfully. No more problems.
